README.md

```diff
@@ -1,12 +1,14 @@
-# jupyter_chat
+# jupyter-chat
 
-[![Github Actions Status](https://github.com/jupyterlab/jupyter-chat/workflows/Build/badge.svg)](https://github.com/jupyterlab/jupyter-chat/actions/workflows/build.yml)[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/jupyterlab/jupyter-chat/main?urlpath=lab)
+[![Github Actions Status](https://github.com/jupyterlab/jupyter-chat/workflows/Build/badge.svg)](https://github.com/jupyterlab/jupyter-chat/actions/workflows/build.yml)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/jupyterlab/jupyter-chat/main?urlpath=lab)
 
 This project is a monorepo containing the frontend components and extensions to build
 a chat in Jupyter.
 
-A lot of the components of this chat project come from
-[jupyter-ai](https://github.com/jupyterlab/jupyter-ai).
+Many components of this chat project come from [jupyter-ai](https://github.com/jupyterlab/jupyter-ai).
+
+![a screenshot showing the jupyter-chat extension used in two browser windows](https://github.com/jtpio/jupyter-chat/assets/591645/977b6f8b-df73-4973-a4f2-abe5abc688c4)
```

Comment:
The link seems to be on a fork of the repo.
Don't you think that we should add the image in this repo ?

Reply:
Thanks for catching this! I updated the link so it points to the repo.

We could also add the screenshot as a real file to the repo, if needed.

Reply:
I don't know if there is a lifetime for this kind of asset. If not, no need to add the file for now.

Reply:
Not sure about lifetime or expiration for these assets (uploaded by drag and dropping them in an issue or comment box). I would guess they don't expire as otherwise they might not show up anymore in issues and PRs in the future?

If it breaks at some point we could add it back as a regular file.